contracts/verifiers/UniversalVerifier.sol

```diff
@@ -21,7 +21,7 @@ contract UniversalVerifier is
     /**
      * @dev Version of contract
      */
-    string public constant VERSION = "1.1.1";
+    string public constant VERSION = "1.1.2";
```

Comment:
You need to bump the version in `contractsInfo` in `constants.ts` as well. That is used for version monitoring scripts